salt/environment_settings.yml

```diff
@@ -884,6 +884,20 @@ environments:
         security_groups:
           - webapp
           - ssh
+  ocw-qa:
```

Comment:
We don't want to add a new environment definition here since it's all still in the same VPC. The environments in this file map directly to separate VPCs for network segmentation which doesn't exist for OCW. We can add the additional purpose for passing the domain names to the QA instance, but we should be matching on a different grain, such as `purpose` or adding a new one like `ocw-env` to differentiate between "QA" and "production" in the OCW context. We don't want to split them into different actual envs, we just want to keep the current architecture as is as much as possible until we cut over from production.

Reply:
Ok, that sounds good. I will follow up with a change along those lines. I think it will probably be to add `ocw-env`, because the property `purposes` exists (as `environments.ocw.purposes`) in `environment_settings.yml` and aligns with the existing `app_name` in the `.sls` file.

I will also add the new grain to the existing minions. I'm making a separate comment in the main thread of this PR so I don't forget.